config/locales/server.en.yml

```diff
@@ -188,7 +188,6 @@ en:
   topic_not_found: "Something has gone wrong. Perhaps this topic was closed or deleted while you were looking at it?"
 
   just_posted_that: "is too similar to what you recently posted"
-  has_already_been_used: "has already been used"
```

Comment:
Hmm looks like this is still being used https://github.com/discourse/discourse/blob/e3e31441b863bd09c51a61e1340673de0edf6488/app/models/topic.rb#L59